include.js

```diff
@@ -97,6 +97,13 @@ let cards = [
     author: 'Ahmed',
     githubLink: 'https://github.com/ahmedalhamad7'
   },
+  {
+    artName: 'DayToNight',
+    pageLink: './Art/NituCStefann/DayToNight.html',
+    imageLink: './Art/NituCStefann/DayToNight.png',
+    author: 'NituCStefann',
+    githubLink: 'https://github.com/NituCStefann'
+  }
```

Comment:
# Feedback - incorrect file links/paths
- your contribution does not have the folder named after your github username
- your links does match the folder structure you've submitted